python-checks/src/main/java/org/sonar/python/checks/cdk/ResourceAccessPolicyCheck.java

```diff
@@ -34,27 +37,37 @@ public class ResourceAccessPolicyCheck extends AbstractIamPolicyStatementCheck {
 
   private static final String MESSAGE = "Make sure granting access to all resources is safe here.";
   private static final String SECONDARY_MESSAGE = "Related effect";
+  private static final Set<String> SENSITIVE_AWS_ACTIONS;
+  private static final String FILEPATH_SENSITIVE_AWS_ACTIONS = "/org/sonar/python/checks/cdk/sensitiveAwsActions.txt";
```

Comment:
I think `sensitiveAwsActions.txt` should be sufficient if you inline `FileResourceUtils.loadResourceAsSet` method